### PR TITLE
Check which EARLY_RESET events could be retried

### PR DESCRIPTION
**Pull request by bemasc**

### CODE REVIEW

File: Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java [hunk 1]
```diff
@@ -123,7 +151,66 @@ public void onError(Pipe pipe, Exception exception) {
...
+    Retry(Context context, SocketAddress dest, ByteBuffer uploadBuffer, Bundle event) {
+      this.context = context;
+      this.dest = dest;
+      this.uploadBuffer = uploadBuffer;
+      this.event = event;
+    }
+
+    private boolean retryTest() {
+      try {
+        Socket socket = new Socket();
```
Comment (fortuna) [inline Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java]: Let's always close this socket

File: Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java [hunk 2]
```diff
@@ -123,7 +151,66 @@ public void onError(Pipe pipe, Exception exception) {
...
+        new Thread(new Retry(context, getAddress(pipe), uploadBuffer, event)).start();
+      }
+    }
+  }
+
+  /**
+   * Class for testing whether an unreachable server becomes reachable if we reduce the size of
+   * the initial packet.
+   */
+  private static class Retry implements Runnable {
```
Comment (fortuna) [inline Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java]: Maybe name it SplitTest or something that indicates this is a side test

File: Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java (hunk 1 above)
Comment (bemasc) [reply Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java]: Done

File: Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java (hunk 2 above)
Comment (bemasc) [reply Android/app/src/main/java/app/intra/socks/ReliabilityMonitor.java]: Done

**Comment by bemasc** [comment]
> I don't see the changes. Did you export them?

Hm.  I don't know what the problem could be.  I see them, e.g. here: https://github.com/Jigsaw-Code/Intra/pull/128/commits/ec6229f7f86dd1efff5351e2da009e8bf31a49e3

**Pull request merged**